legal-api/src/legal_api/reports/business_document.py

```diff
@@ -499,6 +501,44 @@ def _set_liquidation_details(self, business: dict):
                 liquidation_info['recordsOffice'] = records_office_info
         business['liquidation'] = liquidation_info
 
+    def _set_continuation_in_details(self, business: dict):
+        """Set continuation in filing data."""
+        continuation_in_info = {}
+        continuation_in_filing = Filing.get_filings_by_types(self._business.id, ['continuationIn'])
+        if continuation_in_filing:
+            continuation_in_filing = continuation_in_filing[0]
+            jurisdiction = Jurisdiction.get_continuation_in_jurisdiction(continuation_in_filing.business_id)
+
+            # Format country and region
+            region_code = jurisdiction.region
+            country_code = jurisdiction.country
+            country = pycountry.countries.get(alpha_2=country_code)
+            if region_code and region_code.upper() != 'FEDERAL':
+                region = pycountry.subdivisions.get(code=f'{country_code}-{region_code}')
+            region_name = region.name.upper() if region else country.name
```

Comment:
fixing this now, should be else ''

Reply:
refer: https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/reports/report.py#L608

Reply:
refer: https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/reports/business_document.py#L434